sql/binlog.cc

```diff
@@ -8395,26 +8372,16 @@ int MYSQL_BIN_LOG::prepare(THD *thd, bool all) {
     active transactional storage engines, such as is the case if this is a
     replication applier and log_slave_updates=0.
   */
-<<<<<<< HEAD
 #ifdef WITH_WSREP
-  DBUG_ASSERT(thd->wsrep_applier ||
+  assert(thd->wsrep_applier ||
               (thd->slave_thread ? opt_log_slave_updates
```

Comment:
This doesn't look it is clang formatted.

Reply:
fixed.